Mathlib/Geometry/Manifold/Diffeomorph.lean

```diff
@@ -195,6 +195,7 @@ theorem coe_refl : ⇑(Diffeomorph.refl I M n) = id :=
 end
 
 /-- Composition of two diffeomorphisms. -/
+@[trans pp_dot]
```

Comment:
Is the `pp_dot` here intended?

Reply:
I think I missed removing this one. Let's just throw the question of `pp_dot` to some future PR.